packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1761,8 +1762,8 @@ export default class Meeting extends StatelessWebexPlugin {
       LOCUSINFO.EVENTS.CONTROLS_MEETING_TRANSCRIBE_UPDATED,
       ({caption, transcribing}) => {
         // @ts-ignore - config coming from registerPlugin
-        if (transcribing && this.transcription && this.config.receiveTranscription) {
-          this.receiveTranscription();
+        if (transcribing && !this.transcription && this.config.receiveTranscription) {
```

Comment:
shouldn't a test be changed for this logic flip? i.e., `this.transcription -> !this.transcription` ?

Reply:
I checked but there's no test written for this if condition. For the new piece of logic which I have added as part of this PR, I have added a test case for it.

Reply:
But if you changed the logic here... there should be a test for this if condition (unless it's not feasible to do so for some reason?) In the tests below - I don't see anything for checking with the !this.transcription, it's all solely based off of the config.

Reply:
Added tests for locus event `CONTROLS_MEETING_TRANSCRIBE_UPDATED` as well for both if and else condition.